### PR TITLE
Update 03-generators.mdx

### PR DESCRIPTION
**Pull request by EndyKaufman**
Append new generator for NestJS

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx [hunk 1]
```diff
@@ -69,6 +69,7 @@ The following is a list of community created generators. If you want to create y
 - [`prisma-docs-generator`](https://github.com/pantharshit00/prisma-docs-generator): Generates an individual API reference for Prisma Client
 - [`prisma-json-schema-generator`](https://github.com/valentinpalkovic/prisma-json-schema-generator): Transforms the Prisma schema in [JSON schema](https://json-schema.org/)
 - [`typegraphql-prisma`](https://github.com/MichalLytek/typegraphql-prisma#readme): Generates [TypeGraphQL](https://typegraphql.com/) CRUD resolvers for Prisma models
+- [`typegraphql-prisma-nestjs`](https://github.com/EndyKaufman/typegraphql-prisma-nestjs#readme): Fork of Generates [TypeGraphQL](https://github.com/MichalLytek/typegraphql-prisma) CRUD resolvers for Prisma models but for NestJS
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]:
```suggestion
- [`typegraphql-prisma-nestjs`](https://github.com/EndyKaufman/typegraphql-prisma-nestjs#readme): Fork of [TypeGraphQL](https://github.com/MichalLytek/typegraphql-prisma), which also generates CRUD resolvers for Prisma models but for NestJS
```

**Comment by EndyKaufman** [comment]
@janpio

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx [hunk 2]
```diff
@@ -70,6 +70,7 @@ The following is a list of community created generators. If you want to create y
 - [`prisma-json-schema-generator`](https://github.com/valentinpalkovic/prisma-json-schema-generator): Transforms the Prisma schema in [JSON schema](https://json-schema.org/)
 - [`prisma-json-types-generator`](https://github.com/arthurfiorette/prisma-json-types-generator): Adds support for [Strongly Typed `Json`](https://github.com/arthurfiorette/prisma-json-types-generator#readme) fields for all databases. It goes on `prisma-client-js` output and changes the json fields to match the type you provide. Helping with code generators, intellisense and much more. All of that without affecting any runtime code.
 - [`typegraphql-prisma`](https://github.com/MichalLytek/typegraphql-prisma#readme): Generates [TypeGraphQL](https://typegraphql.com/) CRUD resolvers for Prisma models
+- [`typegraphql-prisma-nestjs`](https://github.com/EndyKaufman/typegraphql-prisma-nestjs#readme): Fork of [TypeGraphQL](https://github.com/MichalLytek/typegraphql-prisma), which also generates CRUD resolvers for Prisma models but for NestJS
```
Comment (jharrell) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]:
```suggestion
- [`typegraphql-prisma-nestjs`](https://github.com/EndyKaufman/typegraphql-prisma-nestjs#readme): Fork of [`typegraphql-prisma`](https://github.com/MichalLytek/typegraphql-prisma), which also generates CRUD resolvers for Prisma models but for NestJS
```

**Review by jharrell — CHANGES_REQUESTED** [review]
Sorry to nitpick, but let's be a bit more specific and say that this is a fork of `typegraphql-prisma`

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx (hunk 2 above)
Comment (EndyKaufman) [reply content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]: ok

**Review by jharrell — APPROVED** [review]

**Comment by EndyKaufman** [comment]
@ruheni

**Pull request merged**